content/before-you-fly/code-of-conduct.md

```diff
@@ -6,46 +6,45 @@ weight: 2
 featured_image: "/img/become-a-member.jpg"
 ---
 
-The Blue Mountains Soaring Club thanks you for your membership contribution. We also welcome your ideas and the giving of your time to club meetings and projects. Your club supports your access to sites but needs your support in return.
+The Blue Mountains Soaring Club (BMSC) thanks you for your membership contribution. We also welcome your ideas and the giving of your time to club meetings and projects. Your club supports your access to sites but **needs your support in return**.
 <!--more-->
 
-The Blue Mountains Soaring Club is an affiliate member of the [Sports Aviation Federation of Australia (SAFA)](https://www.safa.asn.au/) and as such the BMSC adopts the SAFA Code of Conduct. SAFA Code of Conduct can be found at; https://members.safa.asn.au/isonic-downloaddoc.php?docid=0X0X1X1X47Wnkvb2IwaDFScW9iOU9nVi9Ody9sdz09
+The Blue Mountains Soaring Club is an affiliate member of the [Sports Aviation Federation of Australia (SAFA)](https://www.safa.asn.au/) and as such the **BMSC adopts the SAFA Code of Conduct**. You can [read the SAFA Code of Conduct on their website](https://members.safa.asn.au/isonic-downloaddoc.php?docid=0X0X1X1X47Wnkvb2IwaDFScW9iOU9nVi9Ody9sdz09).
```

Comment:
Main change here was to reword the second sentence so I could make the link text for the safa CoC an action.

Also note that the `[Sports Aviation Federation of Australia (SAFA)](https://www.safa.asn.au/)` is turned into a link when the website is published as you'll see on the staging site. So only the text in the square brackets is displayed and it links to the address in the curved braces. (Same below).